src/app_service/service/community_tokens/service.nim

```diff
@@ -1392,6 +1398,8 @@ QtObject:
       )
       self.threadpool.start(arg)
       return
+    else:
+      debug "Unknown tokenType received ", token=communityToken.tokenType
```

Comment:
can you make the message more related? Eg. "can't fetch holders for unknown token type", or sth like this

Reply:
Fixed